api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/NewClaimService.java

```diff
@@ -67,8 +67,8 @@ private Claim createAndSaveClaim(Claimant claimant, EligibilityAndEntitlementDec
         return claim;
     }
 
-    private void sendNewCardMessage(Claim claim, PaymentCycleVoucherEntitlement voucherEntitlement) {
-        NewCardRequestMessagePayload payload = buildNewCardMessagePayload(claim, voucherEntitlement);
+    private void sendNewCardMessage(Claim claim, EligibilityAndEntitlementDecision decision) {
+        NewCardRequestMessagePayload payload = buildNewCardMessagePayload(claim, decision.getVoucherEntitlement(), decision.getDateOfBirthOfChildren());
         messageQueueDAO.sendMessage(payload, CREATE_NEW_CARD);
```

Comment:
This is the third variation of children's DOB list: `childrenDob`, `dateOfBirthOfChildren` and `datesOfBirthOfChildren` - should we decide on a standard?

Reply:
`childrenDob` is persisted in PaymentCycle so changing that is a bit more cumbersome. but changing `dateOfBirthOfChildren` to `datesOfBirthOfChildren` is a good plan.